typescript2/textmate-grammar/baml.tmLanguage.yaml

```
@@ -0,0 +1,772 @@
+
+  # A for loop, which can either be a C-style for loop with an initializer, condition, and increment, or an enhanced for-in loop.
+  for-loop:
+    begin: \b(for)\b
+    beginCaptures:
+      "1":
+        name: keyword.control.baml
+    end: (?<=\})
+    patterns:
+      - include: "#for-in-clause"
+      - include: "#imperative-block"
+      - include: "#statement" # Match any statement to allow for C-style for loops
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)